### DISCUSSION TITLE
Build time hook

### DISCUSSION THREAD

**Discussion by mitraecp** [nuxt/nuxt#27040] [Questions]
If possible, how can i create a plugin to execute something in build time? 
i want to change a value in a component just on build time.

**Comment by naseefpv** [comment]
Hi, you can refer the guide from here [Hooks](https://nuxt.com/docs/guide/going-further/hooks) & [List of Hooks](https://nuxt.com/docs/api/advanced/hooks). Hope this helps.

**Reply by hecktarzuli** [reply]
+1 for using a hook.  You'd likely want to set a variable in your Runtime Configs, reference that value in your component and then modify it at build time.  I believe we did something similar in Nuxt 2 where we baked the build date/time into our DOM template as an attribute.  This would likely be similar.
